website/src/components/MultiDimensionalContentWidget.js

```diff
@@ -77,6 +77,8 @@ export const MultiDimensionalContentWidget = () => {
 			selectedNetwork = "Goerli-Prater";
 		else if (isSelectedByText('Sepolia'))
 			selectedNetwork = "Sepolia";
+    else if (isSelectedByText('Holesky'))
```

Comment:
this must be the funky tab vs spaces piece

Reply:
yep, any idea how to format this properly? I can go replace these with tabs, but looking for a programmatic way if possible

Reply:
Fixed it in github UI